homeassistant/config_entries.py

```diff
@@ -1045,7 +1045,7 @@ class FlowCancelledError(Exception):
     """Error to indicate that a flow has been cancelled."""
 
 
-class ConfigEntriesFlowManager(data_entry_flow.FlowManager[ConfigFlowResult]):
+class ConfigEntriesFlowManager(data_entry_flow.FlowManager[ConfigFlowResult, str]):
```

Comment:
```suggestion
class ConfigEntriesFlowManager(data_entry_flow.FlowManager[ConfigFlowResult]):
```
Same here and all other cases in this file.